packages/generators/utils/entry.ts

```diff
@@ -25,8 +25,9 @@ export default function entry(self: IEntry, answer: {
 			.prompt([
 				InputValidate(
 					"multipleEntries",
-					"Type the names you want for your modules (entry files), separated by comma [example: app,vendor]",
```

Comment:
don't remove examples

Reply:
I've removed them in favor of default values:

![multixd](https://user-images.githubusercontent.com/11232797/55822019-08a98b00-5aff-11e9-8875-07f284db64e7.png)